### ISSUE TITLE
Navigation Menu: Setup progress bar missing from My Home menu item

### ISSUE THREAD

**Issue by p-jackson** [Automattic/wp-calypso#52630]
### Steps to reproduce the behavior

1. Head to Calypso on a site that hasn't finished the onboarding checklist (the one on the `/home` view)
2. Notice the progress bar that used to appear in the My Home menu item is no longer there
   ![](https://user-images.githubusercontent.com/1500769/117221551-992d3380-ae5d-11eb-9d53-6fe2a0b382c3.png)
3. Add the `?disable-nav-unification` query param to the url and reload
4. See the progress bar appears when nav unification is disabled
   ![](https://user-images.githubusercontent.com/1500769/117221620-c4178780-ae5d-11eb-8db5-60b6364cc7fe.png)

The code for displaying the progress bar still exists in Calypso:
https://github.com/Automattic/wp-calypso/blob/5e0a07e0ddaff970249d36123ff90e940bbacee0/client/my-sites/sidebar/index.jsx#L290

But it appears to be overridden by nav-uniification.

The sparkline in the Stats menu item still exists, so it appears at least possible for these little widgets to work with nav-unification.

#### Level of impact (Does it block purchases? Does it affect more than just one site?)

No impact for customers who have launched their sites, but new users could be having more trouble launching their sites because there isn't the visual prompt in the menu to complete their setup.

**Comment by jordesign** [comment]
I'm not aware of any user enquiries regarding this. It seems it wouldn't hurt to have it back - but pretty low priority. Shifting this to 'future'.

**Comment by p-jackson** [comment]
IMO this doesn't really feel like part of the My Home pod work and is more a bug that needs fixed as part of Nav Unification. @mmtr I see you added it to the "Back to Basics: My Home Pod" project, do you feel differently? Or perhaps @simison has an opinion on whether this is in scope for us.

**Comment by mmtr** [comment]
@p-jackson Correct, this is a Nav Unification issue. I was under the impression that those issues are now spread across different teams, so since the issue is specific to My Home, I thought it was a good fit for the My Home pod.

For context, I'm currently on an on-call rotation and as part of my duties I triage Nav Unification issues, which is why I moved the issue to a different project. If you think this out of the scope of the pod, I'm happy to move it back to the Dotcom Manage backlog so it can be prioritized after the Back to Basics initiative.

**Comment by p-jackson** [comment]
> I was under the impression that those issues are now spread across different teams, so since the issue is specific to My Home, I thought it was a good fit for the My Home pod.

You could be right, I'll wait for @simison to chime in. Thanks!

**Comment by simison** [comment]
> I'm happy to move it back to the Dotcom Manage backlog so it can be prioritized after the Back to Basics initiative.

@p-jackson @autumnfjeld  @jeffgolenski  I think it's better My home pod owns this. It's important you consider the flows around my home as well, and not only the square visible area of my home. That might bring you to work in the editor, or in the navigation, for example.

**Comment by Copons** [comment]
With the Nav Unification works resuming, we can take care of this.

The unified sidebar is capable of showing this kind of data (although please note that the items are shorter than in the Calypso sidebar, so we have less room for label + progress bar), and the checklist status comes from an endpoint (see [data layer](https://github.com/Automattic/wp-calypso/blob/10998ea1e5de4f2c6afc81061fbbfd7fceecd967/client/state/data-layer/wpcom/sites/launch/index.js)), so we should have access to it from anywhere.

**Comment by obenland** [comment]
@simison This has been defunc'd for 6+ months. Before we go ahead and duplicate that component in PHP, is this still needed to be in the sidebar? Have the other improvements we've made on launch flows made this obsolete?

**Comment by mmtr** [comment]
> Have the other improvements we've made on launch flows made this obsolete?

I took a look at the trends of the `wpcom_site_launched` event over the last couple of years and it seems that it has not been affected by the lack of the progress bar introduced by the Nav Unification release.

Nav Unification was released in March 2021, and although the number of site launches we had in April and May was lower than in March, the monthly trend looks better and/or similar to the ones we had between June 2020 and February 2021. There is a significant drop in June 2021, but that's likely unrelated to Nav Unification since the feature was already live for 3 months.

Furthermore, the progress bar was introduced in May 2020 (https://github.com/Automattic/wp-calypso/pull/42022), but the site launches trend shows a decline in the following months (between June and July), so apparently it didn't help either to encourage our customers to launch their sites.

I may be totally wrong with my analysis though, since there are many factors that could've affected the trends, but overall it seems the progress bar doesn't add too much value, so I don't think it's worth bringing it back.

**Comment by simison** [comment]
Let's hold on with this; we need design team to look how we might do one-click entries to "my home" / "launchpad" / "help center" and progress bar like this should be part of the considerations.

**Comment by simison** [comment]
Thank you @mmtr for analysis. I wouldn't except movements in high-level metric like site launch, and UI choices like this mostly impact how confident people feel when navigating around our interfaces, and how clear our navigations are.

**Comment by youbrokesomething** [comment]
@p-jackson I'm rescuing tickets that have been abandoned by team/priority changes. Is this something we still want to work on? If so.. what team should it go to?
